scripts/verify_genproto.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# This scripts is automatically run by CI to prevent pull requests missing running genproto.sh
+# after changing *.proto file.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+tmpWorkDir=$(mktemp -d -t 'twd.XXXXXX')
+mkdir "$tmpWorkDir/etcd"
+tmpWorkDir="$tmpWorkDir/etcd"
+cp -r . "$tmpWorkDir"
```

Comment:
Coping whole repo seems like a lot of work. Could we maybe just copy `.proto` files?

Reply:
I tried it, but the genproto.sh requires too many files (not just `*.proto`).

Reply:
Could we then simplify genproto.sh so it can be run on non local directory?

Reply:
It is not so easy, as the genproto.sh requires some Go files, extra shell scripts in the scripts directory, the Documentation directory, and so on.

Reply:
Ok, let's leave for future.